lib/index.ts

```diff
@@ -86,9 +94,25 @@ class PWMetrics {
         await sheets.appendResults(results.runs);
       }
     }
+
+    if (resultHasExpectationErrors && this.flags.expectations) {
+      throw new Error(messages.getMessage('HAS_EXPECTATION_ERRORS'));
```

Comment:
@paulirish didn't we wanna CI continue its work and don't throw on found expectations?